class-20.md

```diff
@@ -0,0 +1,48 @@
+# Intro to Blockchain Development
+
+### Class 20 - Other Blockchains & Consensus
+
+We've reviewed some basic blockchain implementations, now we will look into some alternative consensus mechanisms. 
+
+Reading & Videos:
+
+- [DPOS - Dan Larimer](https://steemit.com/dpos/@dantheman/dpos-consensus-algorithm-this-missing-white-paper) - Dan Larimer (author of graphene chains bitshares, eos and steem) posting about consensus in DPOS
+- [Hackernoon: Traditional vs Delegated Proof of Stake](https://hackernoon.com/the-difference-between-traditional-and-delegated-proof-of-stake-36a3e3f25f7d) - A comparison between traditional and delegated proof of stake solutions
+- [Tendermint](https://tendermint.com/docs/introduction/introduction.html) - A short article on tendermint consensus
+
+### Practical Byzantine Fault Tolerance
+
+A proof of stake system offers some efficiency improvements and financial incentives outside of a simple proof of work system to mine cryptocurrencies. However, it often doesn't solve scaling challenges fully. It is difficult to validate a set of data fully across an entire network at the levels required by some modern applications. To mitigate this, a select set of high performance nodes perform validation on behalf of the network, and share data to other parties.
+
+### Tendermint
+
+Tendermint is an implementation of a dBFT blockchain. It can use a cryptocurrency optionally, or be used simply as a database. You can check a very popular implemenation of it in the Smart Cosmos network. A rough diagram of the implementation can be seen below:
```

Comment:
> It can use a cryptocurrency optionally, or be used simply as a database.

It can optionally use a cryptocurrency (for what?) or simply be used as a database.